docs/source/agenda_based.rst

```diff
@@ -4,20 +4,25 @@ Agenda-based simulator
 In the agenda-based simulator, the dialogues are simulated by following the principle of Markov Decision Processes. That is, dialogues are seen as sequences of state transitions enabled by intents (or dialogue actions) and states are only dependent on the previous state. 
 However, as the agenda-based simulator depends on past observations (i.e., annotated dialogues), it is technically described by a *partially observable markov decision process* (POMDP).
 
-Specifically, the simulator's next action is decided by the agent response: if the agend responds expectedly, the next action would the the top action in the agenda, otherwise the simulator will repeat itself.
+Specifically, the simulator's next action is decided by the agent response.
+If the agent responds expectedly, the next action is the top action in the agenda, otherwise the simulator estimates the next action based on the agent intent distribution.
 
-Response generation
--------------------
+Agenda initialization
+---------------------
 
-The response generation module consists of three main steps:
+Before the initialization of the agenda, the simulator needs to know the distribution of intents in the annotated dialogues.
+An intent distribution for each participant (i.e., user and agent) is created.
+The user intent distribution considers successive user intent pairs, i.e., the intent of the first user utterance and the one from the next user utterance.
+The agent intent distribution considers successive agent and user intent pairs, i.e., the intent of the first agent utterance and the intent of the next user utterance.
```

Comment:
italicize *user intent distribution* and *agent intent distribution*

Reply:
Not relevant anymore.